doc/developer-guides/hld/hld-devicemodel.rst

```diff
@@ -110,7 +109,7 @@ Here's an example showing how to run a VM with:
      -s 1:0,lpc -l com1,stdio \
      -s 5,virtio-console,@pty:pty_port \
      -s 3,virtio-blk,/home/acrn/UserVM.img \
-     -s 4,virtio-net,tap_LaaG \
+     -s 4,virtio-net,tap=LaaG \
```

Comment:
@weichenli-intel For my education, what does `tap=LaaG` mean? Line 102 says "Virtio-net device on PCI 00:04.0" -- is LaaG a device name and where is the LaaG device defined (i.e., how does LaaG correspond to PCI 00:04:0)?